changelogs/current.yaml

```diff
@@ -63,6 +63,9 @@ new_features:
 - area: upstream
   change: |
     added :ref:`maglev extension <envoy_v3_api_msg_extensions.load_balancing_policies.maglev.v3.Maglev>` to suppport the :ref:`load balancer policy <envoy_v3_api_field_config.cluster.v3.Cluster.load_balancing_policy>`.
+- area: tls
+  change: |
+    added support to configure the ``enforce_rsa_key_usage`` of SSL_CONFIG. This can be used to override its configuration in BoringSSL.
```

Comment:
Please ref link to the new configuration option. Also duplicate the text that I suggested for the warning on configs that would fail.